labellab_mobile/lib/screen/project/upload_image/project_upload_image_screen.dart

```diff
@@ -204,7 +204,7 @@ class ProjectUploadImageScreen extends StatelessWidget {
         Provider.of<ProjectUploadImageBloc>(context, listen: false)
             .getImageIndex(image);
 
-    File? editedFile = await ImageCropper.cropImage(
+    File? editedFile = await ImageCropper().cropImage(
```

Comment:
Don't initialize ImageCropper here itself.
Create an instance at the top level of the widget.

```
class ProjectUploadImageScreen extends StatelessWidget {

  final _imageCropper = ImageCropper();
```

Then use that here.

```
File? editedFile = await _imageCropper.cropImage(
```